app/assets/stylesheets/global.css.sass

```diff
@@ -2,169 +2,130 @@
 //
 // Site-wide used styles.
 
-*
-  box-sizing: border-box
+// For debugging purposes
+// *
+//   border: 1px solid blue
 
-body
-  z-index: 0
-  position: relative
-  margin: 0 auto
-  padding-bottom: 5em
+html, body
   color: $main
   font-family: Lato, sans-serif
   font-size: 15px
-  a
+  // For sticky footer
+  height: 100%
+
+  a, a:hover
     color: $railsred
-  a:hover
     color: $railsred
-  &:before
-    z-index: -1
-    position: absolute
-    left: 0
-    right: 0
-    top: 0
-    height: 66px
-    background-color: $railsred
-    content: ' '
-
-body>header,
-#content
-  width: 1026px
-  margin: 0 auto
-
-#footer
-  footer
-    border-top: 1px solid $lightgray
-    position: relative
-    top: 3em
-    width: 1026px
-    margin: 0 auto
-
-  ul
-    padding-left: 0.5em
-
   li
-    display: inline-block
-    margin: 1em 0 0 0
-
-    &:last-child:after
-      content: none
-
-    &:after
-      content: ''
-      display: inline-block
-      width: 1em
-      font-family: FontAwesome
-      line-height: 1em
-      color: $lightgray
-
-    a
-      display: inline-block
-      height: 48px
-      text-decoration: none
-
-li
-  list-style-type: none
-
-
-header
-  h1
-    display: inline-block
-    margin: 0
-    width: 210px
-    height: 58px
-    a
-      display: block
-      height: 100%
-      width: 100%
-      background-image: asset-url('logo_subnavi.png')
-      background-repeat: no-repeat
-      background-size: cover
-      background-position: -15px -1px
-      text-indent: 999%
-      white-space: nowrap
+    list-style-type: none
+  .text-right
+    text-align: right
+
+// For sticky footer
+#wrap
+  min-height: 100%
+  height: auto
+  // Negative indent footer by its height
+  margin: 0 auto -60px
+  // Pad bottom by footer height
+  padding: 0 0 60px
+
+#header
+  height: 60px
+  background-color: $railsred 
+  img
+    width: 205px
+    margin: -3px 0 0 -15px
+
+  a.button-burger
+    @include icon($reorder)
+    width: 35px
+    height: 35px
+    font-size: 24px
+    text-decoration: none
+    color: $railsred
+    background: $white
+    border-radius: 50%
+    overflow: hidden
+    margin-top: 12px
+    @media screen and (min-width: 992px)
+      display: none
+    i
+      position: relative
+      top: 5px
+      right: 7px
+  ul
+    padding: 0
+    margin-top: -15px
+    text-align: right
+    li
+      margin-bottom: 0.5rem
 
-  nav
-    clear: right
-    float: right
-    @include clearfix
+  nav.user
+    color: $darkgray
 
-    &.top
+  nav.top
+    @media screen and (min-width: 992px)
```

Comment:
does it work if you use the `$screen-md` variable in less?

e.g.
```sass
@media screen and (min-with: $screen-md)
```

it's a variable in [bootstrap-sass](https://github.com/twbs/bootstrap-sass/blob/master/assets/stylesheets/bootstrap/_variables.scss#L307) and defaulted at `992px`